pytorch_lightning/trainer/logging.py

```diff
@@ -192,6 +192,10 @@ def reduce_distributed_output(self, output, num_gpus):
             if isinstance(output[k], dict):
                 output[k] = self.reduce_distributed_output(output[k], num_gpus)
 
+            # compute the average of scalars
+            elif isinstance(output[k], list):
+                output[k] = sum(output[k]) / len(output[k])
```

Comment:
is there a case when` len(outputs[k])` can be zero?

Reply:
Yes. For example, if you return an empty list for one of your metrics:
```python
logs = {"acc": []}
```
I know it's not clean but such exceptions are caught in [process_output](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/logging.py#L96)

Reply:
Yes, it will be caught under the generic Exception type, which is bad. The exception handling in process_output is another thing we have to deal with sooner or later I guess.